src/main/java/se/sundsvall/casedata/integration/db/model/MessageEntity.java

```diff
@@ -107,6 +107,9 @@ public class MessageEntity {
 	@Column(name = "viewed", nullable = false)
 	private boolean viewed;
 
+	@Column(name = "internal")
+	private Boolean internal;
```

Comment:
Should this attribute support three values? true/false/null

or should it be only true/false?

Reply:
Yes, it will support true/false/null